CMakeLists.txt

```diff
@@ -123,6 +123,18 @@ else ()
     message (AUTHOR_WARNING "${ISSUE_REPORT_MSG}")
 endif ()
 
+# MSVC symbol visibility and linker behavior is sufficiently different from all
+# other platforms. Here, we ensure tests/tools built as EXE choose symbol
+# visibility appropriate for linking the library, whether static or dynamic.
+if (MSVC)
```

Comment:
Thank you for chiming in! My assumptions were incorrect. I'll revert back to using `_WIN32`